pkg/aws/client.go

```diff
@@ -168,7 +166,16 @@ func (b *ClientBuilder) Build() (result Client, err error) {
 		awsSession:          sess,
 	}
 
-	return
+	_, root, err := getClientDetails(c)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get client details: %v", err)
+	}
+
+	if root {
+		return nil, errors.New("using root account is not recommended, please use IAM account instead")
```

Comment:
Just a small nit, ` an IAM user` is a bit better as far as AWS Terminology goes

Reply:
Great! Thanks, I will update the pr.